packages/@aws-cdk/aws-events/lib/archive.ts

```diff
@@ -64,7 +64,7 @@ export class Archive extends Resource {
   constructor(scope: Construct, id: string, props: ArchiveProps) {
     super(scope, id, { physicalName: props.archiveName });
 
-    let archive = new CfnArchive(this, 'Archive', {
+    let archive = new CfnArchive(this, 'Default', {
```

Comment:
This is unfortunately the wrong way to solve this particular bug. It will destroy everyone's existing archives.

This needs to be fixed by setting `this.node.defaultChild = archive`

Reply:
Sorry! I will fix it